src/utilities/borders.scss

```diff
@@ -3,39 +3,39 @@
 
 /* Add a gray border to the left and right */
 .border-x {
-  border-right: $border !important;
-  border-left: $border !important;
+  border-right: $border-rem !important;
+  border-left: $border-rem !important;
 }
 
 /* Add a gray border to the top and bottom */
 .border-y {
-  border-top: $border !important;
-  border-bottom: $border !important;
+  border-top: $border-rem !important;
+  border-bottom: $border-rem !important;
 }
 
 /* Responsive gray borders */
 @each $breakpoint, $variant in $responsive-variants {
   @include breakpoint($breakpoint) {
     /* Add a gray border on all sides at/above this breakpoint */
-    .border#{$variant}        { border: $border !important; }
+    .border#{$variant}        { border: $border-rem !important; }
     .border#{$variant}-0      { border: 0 !important; }
 
-    .border#{$variant}-top    { border-top: $border !important; }
-    .border#{$variant}-right  { border-right: $border !important; }
-    .border#{$variant}-bottom { border-bottom: $border !important; }
-    .border#{$variant}-left   { border-left: $border !important; }
+    .border#{$variant}-top    { border-top: $border-rem !important; }
+    .border#{$variant}-right  { border-right: $border-rem !important; }
+    .border#{$variant}-bottom { border-bottom: $border-rem !important; }
+    .border#{$variant}-left   { border-left: $border-rem !important; }
 
     .border#{$variant}-top-0    { border-top: 0 !important; }
     .border#{$variant}-right-0  { border-right: 0 !important; }
     .border#{$variant}-bottom-0 { border-bottom: 0 !important; }
     .border#{$variant}-left-0   { border-left: 0 !important; }
 
     // Rounded corners
-    .rounded#{$variant}   { border-radius: $border-radius !important; }
+    .rounded#{$variant}   { border-radius: var(--primer-borderRadius-medium,  $border-radius-2) !important; }
     .rounded#{$variant}-0 { border-radius: 0 !important; }
-    .rounded#{$variant}-1 { border-radius: $border-radius-1 !important; }
-    .rounded#{$variant}-2 { border-radius: $border-radius-2 !important; }
-    .rounded#{$variant}-3 { border-radius: $border-radius-3 !important; }
+    .rounded#{$variant}-1 { border-radius: var(--primer-borderRadius-small,  $border-radius-1) !important; }
+    .rounded#{$variant}-2 { border-radius: var(--primer-borderRadius-medium,  $border-radius-2) !important; }
+    .rounded#{$variant}-3 { border-radius: var(--primer-borderRadius-medium,  $border-radius-3) !important; }
```

Comment:
Should this be `--primer-borderRadius-large`? With feature flag enabled it would increase `8px` -> `12px`.